### PR TITLE
Add Marketstack adapter

### PR DESCRIPTION
**Pull request by boxhock**

### CODE REVIEW

File: marketstack/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Input Params
+
+- `base`, `from`, or `coin`: The symbol of the currency to query
+- `endpoint`: Optional endpoint param
+- `interval`: The interval for the data (default: 1min)
+
+## Output
+
+```json
```
Comment (thodges-gh) [inline marketstack/README.md:11]: Don't forget to update the output.

Comment (boxhock) [reply marketstack/README.md:12]: Yep, did it now

File: marketstack/adapter.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+const customParams = {
+  base: ['base', 'from', 'asset'],
+  endpoint: false,
+  interval: false
+}
+
+const createRequest = (input, callback) => {
+  const validator = new Validator(callback, input, customParams)
+  const jobRunID = validator.validated.id
+  const endpoint = validator.validated.data.endpoint || 'intraday'
```
Comment (thodges-gh) [inline marketstack/adapter.js:15]: Maybe introduce a `DEFAULT_ENDPOINT` to follow the convention you've made here?

Comment (boxhock) [reply marketstack/adapter.js:15]: makes sense

File: marketstack/test/adapter_test.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    ]
+
+    requests.forEach(req => {
+      it(`${req.name}`, (done) => {
+        createRequest(req.testData, (statusCode, data) => {
+          assert.equal(statusCode, 200)
+          assert.equal(data.jobRunID, jobID)
+          assert.isNotEmpty(data.data)
+          assert.isAbove(data.result, 0)
+          assert.isAbove(data.data.result, 0)
```
Comment (krebernisak) [inline marketstack/test/adapter_test.js:20]:
```suggestion
assert.isAbove(data.data.result, 0)
assert.equal(data.result, data.data.[0].close)
assert.equal(data.data.result, data.data.[0].close)
```

**Pull request merged**